### PR TITLE
Feature/13 geotiff

### PR DESCRIPTION
**Pull request by drewbo**
Adds geotiff reading support

Adds a very small local tiff for testing. I've also confirmed that it works on the original tif that I took this from: http://oin-hotosm.s3.amazonaws.com/593ede5ee407d70011386139/0/3041615b-2bdb-40c5-b834-36f580baca29.tif (it can be dropped in the "imagery" properties as is; I left this out of the tests because I don't want to ping this every time the tests run)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ Before running any commands, it is necessary to create a `config.json` file to s
...
   - `buffer`: The number of pixels to buffer the geometry by. This is an optional parameter to buffer the label for `object-detection` and `segmentation` tasks. Accepts any number (positive or negative). It uses [Shapely `object.buffer`](https://shapely.readthedocs.io/en/latest/manual.html#object.buffer) to calculate the final geometry. You can verify that your buffer options create the desired labels by inspecting the files created in `data/labels/` after running the `labels` command.
-- `imagery`: A template string for a tiled imagery service. Note that you will generally need an API key to obtain images and there may be associated costs. The above example requires a [Mapbox access token](https://www.mapbox.com/help/how-access-tokens-work/)
+- `imagery`: One of:
+  - A template string for a tiled imagery service. Note that you will generally need an API key to obtain images and there may be associated costs. The above example requires a [Mapbox access token](https://www.mapbox.com/help/how-access-tokens-work/)
+  - A GeoTIFF file location. Works with both local and remote files.
```
Comment (wronk) [inline README.md]: Can you add an example here? The template string is easy to understand from the `config.json` example.

If this ends up getting too complicated, we might want to create a short `Examples` section highlighting the different options.

File: label_maker/images.py [hunk 2]
```diff
@@ -66,9 +64,11 @@ def class_test(value):
     # download tiles
     tiles = class_tiles + background_tiles
     print('Downloading {} tiles to {}'.format(len(tiles), op.join(dest_folder, 'tiles')))
-    o = urlparse(imagery)
-    _, image_format = op.splitext(o.path)
+
+    # get image acquisition function based on imagery string
+    image_function = download_tile_tms
+    if imagery[-4:].lower() == '.tif':
```
Comment (wronk) [inline label_maker/images.py]: Any reason not to use `op.splitext(imagery)[1].lower()` here?

Comment (wronk) [reply label_maker/images.py]: Also, could we ever get `.tiff`?

File: label_maker/utils.py [hunk 3]
```diff
@@ -14,3 +22,38 @@ def class_match(ml_type, label, i):
...
+
+def download_tile_tms(tile, imagery, dest_folder):
+    """Download a satellite image tile from a tms endpoint"""
+    o = urlparse(imagery)
+    _, image_format = op.splitext(o.path)
+    r = requests.get(url(tile.split('-'), imagery))
+    tile_img = op.join(dest_folder, 'tiles', '{}{}'.format(tile, image_format))
+    open(tile_img, 'wb').write(r.content)
+
+def get_tile_tif(tile, imagery, dest_folder):
```
Comment (wronk) [inline label_maker/utils.py]: Add a few more comments within this function

File: label_maker/images.py (hunk 2 above)
Comment (drewbo) [reply label_maker/images.py]: No and yes (I was worried about dots in urls but apparently it works fine). I'll test for both extensions

File: label_maker/utils.py (hunk 3 above)
Comment (wronk) [reply label_maker/utils.py]: Sorry, I meant within the function itself :p

Just 2-3 lines to say what the `proj_to` functions are doing and how the `window`is being used.

**Pull request merged**